cmd/export.go

```diff
@@ -58,18 +88,18 @@ var exportCmd = &cobra.Command{
 			return err
 		}
 		exportType, err := cmd.Flags().GetString("type")
-		exportType = strings.ToLower(exportType)
 		if err != nil {
 			return err
 		}
+		exportType = strings.ToLower(exportType)
+		if validExtension := providedExportTypes[exportType]; !validExtension {
+			return fmt.Errorf("export type not supported")
+		}
```

Comment:
Here you're using the returned key to check if `exportType` exists. Wouldn't it be safer to use the second argument returned? Here's an [example](https://stackoverflow.com/questions/2050391/how-to-check-if-a-map-contains-a-key-in-go)

Reply:
Oh right, that's what I meant to do, my bad